modules/godot/nodes/script_ecs.cpp

```diff
@@ -96,13 +99,15 @@ void ScriptEcs::reload_scripts() {
 		system_bundles[i]->verified = false;
 	}
 
-	// Scan the script classes.
+#ifdef TOOLS_ENABLED
```

Comment:
I think this is not valid, since we still need to load the scripted components. Is `EditorFileSystem` not available in production? We need to find an alternative way of fetching the scripts at production time then 🤔

Reply:
Does not look like. If I run scons with `tools=no` then I get undefined referenced to EditorFileSystem.
I guess these parts of the code are needed outside of the editor?